test/services/twilio-help/twilio-help.test.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Can we change the file name to` twilio-command-help.test.js ` so that it will be sync with the services file.

Reply:
We're testing the behaviour for not just `twilio-command-help` but twilio-help master class [here](https://github.com/twilio/twilio-cli/blob/a938d976bfbefe697984cb15262d4864be2e7301/src/services/twilio-help/custom-help.js#L5-L8) itself. 

`twilio-command-help.test.js` is a subbordinate (Kinda a helper) for the parent class. 
As for the naming for `twilio-help.test.js` (instead of custom-help), deliberately named as the package name since we're testing the overall functionality here. Let me know if you feel otherwise, happy to rename!

Reply:
Make sense. Thanks.